source/TimePicker.js

```diff
@@ -470,10 +492,11 @@
 					if (this.meridiemEnable === true) {
 						this.createComponent(
 							{classes: 'moon-date-picker-wrap', components:[
-								{kind:'moon.MeridiemPicker', name:'meridiem', classes:'moon-date-picker-field', value: valueHours > 12 ? 1 : 0, meridiems: this.meridiems || ['am','pm'], onChange: 'meridiemPickerChanged'},
+								{kind:'moon.MeridiemPicker', name:'meridiem', classes:'moon-date-picker-field', meridiems: this.meridiems || [{name: 'AM', start: '00:00', end: '11:59'}, {name: 'PM', start: '12:00', end: '23:59'}], max: this.meridiems.length - 1 || 1, onChange: 'meridiemPickerChanged'},
 								{name: 'meridiemLabel', content: this.meridiemText, classes: 'moon-date-picker-label moon-divider-text'}
 							]}
 						);
+						this.$.meridiem.value = this.$.meridiem.getMeridiemIndex(values.hour, values.minute);
```

Comment:
Ah right, I guess I was more wondering about the ergonomics and encapsulation here. It seems like we might be able to pass something like an `hour` and a `minute` parameter to `MeridiemPicker` and have it compute its own value internally, rather than set the value of `MeridiemPicker` from another kind, based on this other kind calling a method of `MeridiemPicker` (similarly, on L675). It seems like `getMeridiemIndex` would be more of an internal method for `MeridiemPicker`.